.github/workflows/maven.yml

```diff
@@ -96,3 +101,24 @@ jobs:
           name: pr
           path: pr/
         if: github.event_name == 'pull_request' && matrix.os == 'ubuntu-latest'
+
+  uat-linux:
```

Comment:
let's move this into a separate workflow file please. And set it to run only on pull_request, not on push.